ocamltest/Makefile

```diff
@@ -91,7 +91,6 @@ run := run_$(UNIX_OR_WIN32)
 core := \
   $(run).c \
   run_stubs.c \
-  ocamltest_stdlib_stubs.c \
```

Comment:
Is this really needed ? So far I haven't had any trouble building ocamltest so this surprises me.